### ISSUE TITLE
Class:Event

### ISSUE THREAD

**Issue by magpiedin** [tdwg/cd#362]
|  |  |
| ---- | ---- |
| **Parent** |  |
| **Label** | Event |
| **Definition** | An action that occurs at some location during some time. |
| **Usage** |  |
| **Repeatable** |  |
| **Relationships** |  |
| **Potential standards/vocabularies/ontologies to adopt** |  |
| **Notes** | http://rs.tdwg.org/dwc/terms/version/Event-2018-09-06 |

**Comment by rondlg** [comment]
From December 16th meeting notes:

The important distinction of a Latimer Core record from a Darwin Core record is that it should always make reference to and include event-based groups of occurrences, that is information/evidence that was situated in/taken from space and time. The event can be as simple as a historical approach “all the things [being managed/curated] by a museum at a point in time” or as specific as a particular research project “a monitoring survey of a single animal over a period of time”. However an event doesn’t have to be mandatory in all collection description use cases.

**Comment by jbstatgen** [comment]
see also ABCD class "Gathering": A class to describe a collection or observation event.
https://abcd.tdwg.org/terms/#Gathering
and https://abcd.tdwg.org/xml/documentation/ABCD_3.0/ABCD_3_0_xsd.html#Unit_Gathering

**Comment by tucotuco** [comment]
... and need not refer to gatherings at all.

**Comment by jbstatgen** [comment]
> Note that this is quite specific, whereas a dwc:event can involve many more species, a longer duration and more people.

This sounds as if LC's Event is more in line with ABCD's Dataset(s).

However, with properties "samplingProtocol" and "habitat", those sounded to me more like ABCD's Unit/Gathering/Method: "The method (including technical means) used to make the collection or observation or to describe the capture event." and Unit/Gathering/Biotope: "An element for citing biotope type terms (e.g. classification terms) and measurements (e.g. vegetation height, salinity, slope) related to the unit's gathering site". 

Until now I understood Unit/Gathering as capturing the spatiotemporal, abiotic and biotic (aka "ecological") context from which a collection originated.

**Comment by jbstatgen** [comment]
see misplaced comment [here](https://github.com/tdwg/cd/issues/121#issuecomment-1040638458)



>    This sounds as if LC's Event is more in line with ABCD's Dataset(s).
    I don't know, but that sounds unlikely. An event has a specific time and place, I wouldn't not have thought a dataset would.

 >   However, with properties "samplingProtocol" and "habitat", those sounded to me more like ABCD's Unit/Gathering/Method: 
> "The method (including technical means) used to make the collection or observation or to describe the capture event." and
>    Unit/Gathering/Biotope: "An element for citing biotope type terms (e.g. classification terms) and measurements (e.g. vegetation height, salinity, slope) related to the unit's gathering site".
    Sounds reasonable

Does a mapping exist between abcd:Unit/Gathering and dwc term(s)?
I don't know. It would be best to ask someone from Berlin Botanic Garden, maybe @DavidFichtmueller

**Comment by tucotuco** [comment]
> Does a mapping exist between abcd:Unit/Gathering and dwc term(s)? I don't know. It would be best to ask someone from Berlin Botanic Garden, maybe @DavidFichtmueller

Such a mapping exist on the Darwin Core side in an attribute called abcd_equivalence, which is best seen in the term versions document (https://github.com/tdwg/dwc/blob/master/vocabulary/term_versions.csv), but is also shown for each term on the normative term definitions page (for example, see https://dwc.tdwg.org/list/#dwc_catalogNumber).

**Comment by jbstatgen** [comment]
Thanks @tucotuco 

Following your links (see https://dwc.tdwg.org/list/#dwc_Event)
dwc:Event is equivalent to abcd: DataSets/DataSet/Units/Unit/Gathering

**Comment by tucotuco** [comment]
For these purposes, yes, though the dwc:Event is a broader concept of which
a DataSets/DataSet/Units/Unit/Gathering is one type.

**Comment by DavidFichtmueller** [comment]
I agree, that Gathering is a type of event, but I am not sure about mapping abcd:Gathering to dwc:Event with any property that implies equivalence. 

In ABCD so far we have done mappings using skos:hasExactMatch and skos:hasBroaderMatch, to DwC and other vocabularies. However, this is as well not normative and the semantic implications of this usage are a bit tricky (see https://github.com/tdwg/ac/issues/220 ). We just used it to bring a bit nuance into the mappings, maybe here a mapping using skos:broader (or similar) would be useful.

If anybody is interested, here is a query to show all of the mappings of the classes and properties that are connected to gathering, such as "Measurement Or Fact": [query to show all the mappings](https://wiki.bgbm.org/bdidata/query/#%23this%20query%20shows%20all%20of%20the%20mappings%20from%20any%20concept%20that%20is%20linked%20from%20Gathering%20%28i.e.%20sub%20elements%20in%20the%20XML%20tree%29%20to%20DarwinCore.%0APREFIX%20bwd%3A%20%3Chttp%3A%2F%2Fwiki.bgbm.org%2Fentity%2F%3E%0APREFIX%20bwdt%3A%20%3Chttp%3A%2F%2Fwiki.bgbm.org%2Fprop%2Fdirect%2F%3E%0A%0ASELECT%20DISTINCT%20%3Fconcept%20%3FconceptLabel%20%3Fproperty%20%3FpropertyLabel%20%3Furi%20%20WHERE%7B%0A%20%20%7B%0A%20%20%20%20VALUES%20%3FmappingProperty%20%7Bbwdt%3AP11%20bwdt%3AP12%7D%0A%20%20%20%20%3Fconcept%20bwdt%3AP8%20bwd%3AQ219.%0A%20%20%20%20%23remove%20the%20line%20below%2C%20to%20show%20mappings%20of%20all%20concepts%20and%20not%20just%20the%20ones%20connected%20to%20Gathering%0A%20%20%20%20bwd%3AQ1164%20bwdt%3AP45%2F%28bwdt%3AP13%2F%28bwdt%3AP45%2F%28bwdt%3AP13%29%3F%29%3F%29%3F%20%3Fconcept.%0A%20%20%20%20%3Fconcept%20%3FmappingProperty%20%3Furi.%0A%20%20%20%20%3Fproperty%20wikibase%3AdirectClaim%20%3FmappingProperty.%20%20%0A%20%20%20%20%23remove%20this%20filter%20to%20show%20all%20mappings%2C%20not%20just%20DwC%0A%20%20%20%20FILTER%28STRSTARTS%28STR%28%3Furi%29%2C%20%27http%3A%2F%2Frs.tdwg.org%2Fdwc%2F%27%29%29%0A%20%20%7D%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%0A%7D)

> Until now I understood Unit/Gathering as capturing the spatiotemporal, abiotic and biotic (aka "ecological") context from which a collection originated.

That is how it is used. It is a bit more flexible than the definition Quentin provided from the ICN. For instance an observation can also be expressed using the gathering event, just not with a specimen connected to the unit.

**Comment by tucotuco** [comment]
> If a Unit/Gathering can be associated with, that is, produce a material sample or an information artifact, what else is there that lets dwc:event be broader than abcd:unit/gathering?

A dwc:Event need not generate a material sample or an information artifact. An example from an ecological dataset would be to have a parent-most Event that represents the entire spatiotemporal scope of a project, within which there could be spatiotemporal Events associated with activities at a site, within which may be other nested spatiotemporal subdivisions, within which there might be the kinds of Events you are talking about that produce the material samples, digital media, or or other information artifacts.

**Comment by jbstatgen** [comment]
Ah, agreed. 
I imagine eg. a project or campaign itinerary.

**Comment by essvee** [comment]
Updated the Notes field to clarify how this class is related to dwc:Event.

**Comment by Jegelewicz** [comment]
Are the only events expected expeditions? The properties available appear to point in that direction. I am unsure how this is meant to be used - the records in this collection arose because of this event? The records in this collection were affected by this event? It seems there could be multiple kinds of events of interest (one of them being a change in "status"). If the collection has come about due to hundreds of expeditions should I be expected to add them all?

**Comment by jbstatgen** [comment]
HI @Jegelewicz , good question. 

Rereading our exchange from half a year ago, it seem that we all agree that a dwc:Event can be broader than a typical abcd:Gathering. The reason is that dwc:Event can include events that don't necessarily result in a sample/object/artefact/entity.

As far as I understand it, the expected usage in LtC is that a ltc:Event is something that happens in space and time, and is an activity that relates to an occurrence s. lat., ie. reality out there. Somebody traveling around the world in 80 days is an event (prov:Activity is traveling for 80 days), even if it doesn't result in any collection objects. However, a chance acquaintance that the Verne travelers met during their journey, who collected an entity while talking to them, can refer to the Verne event as an activity related to their collection event.

> If the collection has come about due to hundreds of expeditions should I be expected to add them all?

This seems only practical, if you assemble the list of the associated expeditions by machine action.

> It seems there could be multiple kinds of events of interest (one of them being a change in "status").

Yes, this is how I'm seeing "Event" in the context of the DES and its transactions-based implementation enabled by FAIR Digital Objects (FDOs), cp. DiSSCo. Here also activities in the digital realm are events, ie. open FAIR Digital Objects of type Event (openDEs). Time remains the same, however locality (space) in this case relates to hard- and software. For example a certain CMS software platform on a certain hardware server (standing in some server farm) acting as endpoint (prov:Agent) changes the status (prov:Activity) of a system, digital object or data entity (prov:Entity).

How the concepts for real-world events and digital transactions develop in relationship to each other seems currently open. Maybe they will start to merge, maybe not.
